source/calculus/source/01-LT/03.ptx

```diff
@@ -341,9 +420,25 @@
             Determine the following limits and explain your reasoning.
         </p>
         
-            <task><me>\lim_{x\to-6 } \dfrac{ x^{2} - 6 \, x + 5 }{ x^{2} - 3 \, x - 18 }</me></task>
-            <task><me>\lim_{x\to-1 } \dfrac{ x^{2} - 1 }{ x^{2} + 3 \, x + 2 }</me></task>
-        <task><me>\lim_{x\to5 } \dfrac{ x - 5 }{ \sqrt{x + 31} - 6 }</me></task>
+            <task><me>\lim_{x\to-6 } \dfrac{ x^{2} - 6 \, x + 5 }{ x^{2} - 3 \, x - 18 }</me>
+            <answer>
+              <p>
+                <m>\dfrac{77}{36}</m>
+              </p>
+            </answer></task>
+            <task><me>\lim_{x\to-1 } \dfrac{ x^{2} - 1 }{ x^{2} + 3 \, x + 2 }</me>
+            <answer>
+              <p>
+                <m>-2</m>
+              </p>
+            </answer></task>
+        <task><me>\lim_{x\to5 } \dfrac{ x - 5 }{ \sqrt{x + 31} - 6 }</me>
+         <answer>
+              <p>
+                <m>12</m>
+              </p>
+            </answer>
+        </task>
```

Comment:
This is not your mistake, but we have some tasks inside of a statement that need to be fixed to use this structure:

```xml
<activity>
  <introduction><p>Determine....</p></introduction>
  <task>
    <statement><p><me>...</me></p></statement>
    <answer><p><m>...</m></p></answer>
  </task>
  <task>
    <statement><p><me>...</me></p></statement>
    <answer><p><m>...</m></p></answer>
  </task>
  <task>
    <statement><p><me>...</me></p></statement>
    <answer><p><m>...</m></p></answer>
  </task>
</activity>
```

Reply:
How can I do then? Or, you have fixed it.

Reply:
This needs to be done in your Codespace on the `Solution-to-1.3-Limits-Analytically-(LT3)` branch